android/sdk/src/main/cpp/app/organicmaps/sdk/bookmarks/data/BookmarkManager.cpp

```diff
@@ -447,6 +447,16 @@ JNIEXPORT jobject JNICALL Java_app_organicmaps_sdk_bookmarks_data_BookmarkManage
   return usermark_helper::CreateMapObject(env, g_framework->GetPlacePageInfo());
 }
 
+JNIEXPORT void JNICALL Java_app_organicmaps_sdk_bookmarks_data_BookmarkManager_nativeUpdateTrackPlacePage(JNIEnv * env,
+                                                                                                          jobject,
+                                                                                                          jlong trackId)
```

Comment:
nit:, also, should trackId parameter be removed?:
```suggestion
JNIEXPORT void JNICALL Java_app_organicmaps_sdk_bookmarks_data_BookmarkManager_nativeUpdateTrackPlacePage(JNIEnv *,                                                                                                          jobject,                                                                                                     jlong)
```